pkg/resources/testdata/TestAcc_ExternalVolume/azure/with-encryption-kms-key-id/test.tf

```diff
@@ -0,0 +1,15 @@
+resource "snowflake_external_volume" "complete" {
+  name         = var.name
+  comment      = var.comment
+  allow_writes = var.allow_writes
+  dynamic "storage_location" {
+    for_each = var.storage_location
+    content {
+      storage_location_name = storage_location.value["storage_location_name"]
+      storage_provider      = storage_location.value["storage_provider"]
+      storage_base_url      = storage_location.value["storage_base_url"]
+      azure_tenant_id       = storage_location.value["azure_tenant_id"]
+      encryption_kms_key_id = storage_location.value["encryption_kms_key_id"]
```

Comment:
Nit: Please, use `try(storage_location.value["encryption_kms_key_id"], null)` and similarly for `encryption_type`. This should allow to configure both of these ways in a single test directory.